sdk/samples/iot/hub/src/paho_iot_hub_telemetry_example.c

```diff
@@ -26,13 +26,17 @@
 #include <azure/core/az_span.h>
 #include <azure/iot/az_iot_hub_client.h>
 
+#define TIMEOUT_MQTT_DISCONNECT_MS (10 * 1000)
+#define TELEMETRY_SEND_INTERVAL 1
+#define NUMBER_OF_MESSAGES 5
+
 #ifdef _MSC_VER
 // "'getenv': This function or variable may be unsafe. Consider using _dupenv_s instead."
 #pragma warning(disable : 4996)
 #endif
 
 // DO NOT MODIFY: Device ID Environment Variable Name
-#define ENV_DEVICE_ID "AZ_IOT_DEVICE_ID"
+#define ENV_DEVICE_ID "AZ_IOT_DEVICE_ID_SAS"
```

Comment:
this is a cert sample

Reply:
Yeah - found it myself running the samples. Fixed.